userd/launcher.go

```diff
@@ -40,6 +46,9 @@ const launcherIntrospectionXML = `
 	<method name='OpenURL'>
 		<arg type='s' name='url' direction='in'/>
 	</method>
+	<method name="OpenFile">
+		<arg type="h" name="fd" direction="in"/>
```

Comment:
It would be great if we could prepare for the windowID  here so that we can set the UI transient for the calling thing. But afaict only our xdg-open inside the core snap can figure out the window-id (xdg-open can get the pid of the other side of the dbus connection and then try to find that pid in the active X windows).

Reply:
Ok merge and bind all in one package